### ISSUE TITLE
Docmentation Build Fail

### ISSUE THREAD

**Issue by Cadair** [astropy/ci-helpers#106]
Hi, I don't know if I have missed something, but this is rather annoying:

https://travis-ci.org/sunpy/sunpy/jobs/139200223#L606

```
# All requested packages already installed.
# packages in environment at /home/travis/miniconda/envs/test:
#
astropy                   1.2                 np111py27_0  
numpy                     1.11.0          py27_blas_openblas_200  [blas_openblas]  conda-forge
python                    2.7.11                        0  
Due to a matplotlib issue (#5836), the version for the
            sphinx builds needs to be !=1.5.1. This may override the version
            number specified in 
Fetching package metadata ............
Solving package specifications: ....
The following specifications were found to be in conflict:
  - matplotlib !=1.5.1
  - python 2.7*
Use "conda info <package>" to see the dependencies for each package.
```
it seemed to be working fine before...

**Comment by bsipocz** [comment]
@Cadair - This should be fixed now with #81, please try.

**Comment by Cadair** [comment]
it does seem to be working again. I will double check.
